alacritty/src/renderer/rects.rs

```diff
@@ -328,26 +322,25 @@ impl RectRenderer {
         }
     }
 
+    #[allow(clippy::many_single_char_names)]
     fn add_rect(&mut self, half_width: f32, half_height: f32, rect: &RenderRect) {
         // Calculate rectangle vertices positions in normalized device coordinates.
         // NDC range from -1 to +1, with Y pointing up.
         let x = rect.x / half_width - 1.0;
         let y = -rect.y / half_height + 1.0;
         let width = rect.width / half_width;
         let height = rect.height / half_height;
-        let color = Rgba {
-            r: rect.color.r,
-            g: rect.color.g,
-            b: rect.color.b,
-            a: (rect.alpha * 255.) as u8,
-        };
+        let r = rect.color.r;
+        let g = rect.color.g;
+        let b = rect.color.b;
```

Comment:
```suggestion
        let Rgb { r, g, b } = rect.color;
```

Reply:
whoa, i did not know this is possible. thanks!